samples/k8s_v1_blueprint.yaml

```diff
@@ -9,7 +9,23 @@ metadata:
   name: blueprint-sample
 spec:
   blueprint: |
-    {"blueprintApi":"v2","dogus":[{"name":"official/postgresql","version":"12.15-2","targetState":"present"}]}
+    {
+    "blueprintApi":"v2",
+    "dogus":[
+      {
+        "name":"official/postgresql",
+        "version":"12.15-2",
+        "targetState":"present"
+      }
+    ],
+    "components": [
+      {
+        "distributionNamespace": "k8s",
+        "name":"k8s-dogu-operator",
```

Comment:
Sorry, I think, there has been a misunderstanding. I thought we agreed to let the `distributionNamespace` be a part of the name field, since it's more platform-independent that way.

Reply:
Did you spoke with  @alexander-dammeier about this?, because he introduced the idea.

Reply:
I think we resolved it with 37d5c2dc